scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -530,6 +532,8 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
             File.Delete(compilersDeps);
             File.Delete(compilersRuntimeConfig);
 
+            CrossgenUtil.CrossgenDirectory(c, outputDir);
```

Comment:
I'm fine with this.  I just wanted to call it out as something that may be unnecessary.

As our build keeps getting longer and does more things, we just need to ensure we aren't doing unnecessary things.